oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -2345,11 +2339,6 @@ private boolean withClientSession() {
         return connection.getStatus().isClientSessionSupported() && useClientSession;
     }
 
-    private boolean secondariesWithinAcceptableLag() {
```

Comment:
Is it ok to drop this code?

Reply:
In my opinion yes, I wrote you an email on 2 october providing my opinion about that matter, I copy here what I wrote in that email:

I would like to share with you that the code in MongoDocumentStore like this:

```
if (secondariesWithinAcceptableLag()) {
    dbCollection = getDBCollection(collection);
} else {
    lagTooHigh();
    dbCollection = getDBCollection(collection).withReadPreference(ReadPreference.primary());
}
```

will be replaced simply with this:

`dbCollection = getDBCollection(collection);`

because mongodb-driver-sync knows itself if ask to primary or secondary based on [maxStalenessSeconds property ](https://www.mongodb.com/docs/drivers/java/sync/current/fundamentals/connection/connection-options/)

ReplicaSetStatus class too has not much more sense I think, for sure its method updateLag

Have you some opinion you want to share?

Reply:
Based on the documentation provided, it seems ok to remove this code, as the driver now does this check internally.

Reply:
> Have you some opinion you want to share?

Nope, I'm not a MongoDB expert, it just seemed to be suspicious.